src/bufferutils.js

```diff
@@ -174,8 +119,8 @@ module.exports = {
   readUInt64LE: readUInt64LE,
   readVarInt: readVarInt,
   reverse: require('buffer-reverse'),
-  varIntBuffer: varIntBuffer,
-  varIntSize: varIntSize,
+  varIntBuffer: varuint.encode,
+  varIntSize: varuint.encodingLength,
```

Comment:
@fanatid its worth asking if this should be included.
I'd say these functions are non-generic enough that it makes sense to keep them in this library.